src/plugins/FileService/components/Uploaded.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, Grid, makeStyles, Typography } from '@material-ui/core'
+import React from 'react'
+import { useHistory, useLocation } from 'react-router'
+import { Image } from '../../../components/shared/Image'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { getUrl } from '../../../utils/utils'
+import { useExchange } from '../hooks/Exchange'
+import type { FileInfo } from '../types'
+import { formatDateTime, formatFileSize } from '../utils'
+import { FileName } from './FileName'
+
+const useStyles = makeStyles({
+    container: {
+        height: 250,
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        userSelect: 'none',
+        paddingTop: 18,
+        paddingBottom: 18,
+    },
+    name: {
+        fontSize: 16,
+        lineHeight: 1.75,
+        color: '#3B3B3B',
+        textAlign: 'center',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        width: 400,
+        whiteSpace: 'nowrap',
+    },
+    meta: {
+        fontSize: 14,
+        lineHeight: 1.71,
+        color: '#5D6F88',
+        textAlign: 'center',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+    },
+    info: {
+        margin: 0,
+    },
+    back: {
+        color: '#2CA4EF',
+    },
+})
+
+export const Uploaded: React.FC = () => {
+    const { t } = useI18N()
+    const classes = useStyles()
+    const history = useHistory()
+    const { onInsert } = useExchange()
+    const { state } = useLocation<FileInfo>()
+    React.useEffect(() => {
+        onInsert(state)
+    }, [onInsert, state])
+    const onBack = () => {
+        onInsert(null)
+        history.push('/upload')
+    }
+    const onPreview = (event: React.MouseEvent) => {
+        // ! THIS METHOD IS ONLY IN THE DEBUGGER !
```

Comment:
Is this going to be removed?

Reply:
may be not.

@neruthes

Reply:
Can someone offer some background?

Reply:
Ok. Let this hidden feature remain. It should be useful in certain scenarios while publicizing it is unnecessary.